examples/misc/raptor.py

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env python3
+"""Demonstrate the "raptor" features for remote Task management.
+
+This script and its supporting files may use relative file paths. Run from the
+directory in which you found it.
+
+Refer to the ``raptor.cfg`` file in the same directory for configurable run time details.
+
+By default, this example uses the ``local.localhost`` resource with the ``local``
+access scheme. Before running, confirm that you are able to ssh to localhost
+without being prompted for a password.
```

Comment:
Don't need to ssh with the `local` scheme.

```suggestion
access scheme.
```

Reply:
correct - I removed that sentence now.